features/doc.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Generate documentation for a specific target using Tuist
+
+  Scenario: The project is an application with frameworks (ios_app_with_frameworks)
+    Given that tuist is available
+    And I have a working directory
+    Then I copy the fixture ios_app_with_frameworks into the working directory
+    Then tuist generates the documentation for 'Framework1'
```

Comment:
I'd turn this one into:
```
Then I should be able to access the documentation of target X
```
What that step would do is:
- Run `tuist doc X`.
- Send an HTTP request to localhost:PORT and get the response (i.e. `index.html`).
- If the response is 200, the test succeeds. Then we send a SIGINT signal to the process and finish the test.
- Otherwise we fail the test including the status code that we got from the server.

That's the workflow most users will go through and therefore needs to be captured in our acceptance test.

Reply:
Done 🚀